[CI] Test OKD Bundle GitHub Action workflow failing

### Component

None

### Task description

### Description
I see that Test OKD Bundle GitHub action is always failing.

Looking into the logs it seems, a package requirement is failing:

```shell
level=info msg="Using bundle path /home/runner/.crc/cache/crc_okd_libvirt_4.15.0-0.okd-2024-02-23-163410_amd64.crcbundle"
level=info msg="Checking if running as non-root"
level=info msg="Checking if running inside WSL2"
level=info msg="Checking if crc-admin-helper executable is cached"
level=info msg="Checking if running on a supported CPU architecture"
level=info msg="Checking if crc executable symlink exists"
level=info msg="Checking minimum RAM requirements"
level=info msg="Check if Podman binary exists in: /home/runner/.crc/bin/oc"
level=info msg="Checking if Virtualization is enabled"
level=info msg="Checking if KVM is enabled"
level=info msg="Checking if libvirt is installed"
level=info msg="Checking if user is part of libvirt group"
level=info msg="Checking if active user/process is currently part of the libvirt group"
level=info msg="Checking if libvirt daemon is running"
level=info msg="Checking if a supported libvirt version is installed"
level=info msg="Checking if crc-driver-libvirt is installed"
level=info msg="Checking crc daemon systemd socket units"
level=info msg="Checking if AppArmor is configured"
level=info msg="Checking if vsock is correctly configured"
level=info msg="Loading bundle: crc_okd_libvirt_4.15.0-0.okd-2024-02-23-163410_amd64..."
level=info msg="Creating CRC VM for OKD 4.15.0-0.okd-2024-02-23-163410..."
level=info msg="Generating new SSH key pair..."
level=info msg="Generating new password for the kubeadmin user"
level=info msg="Starting CRC VM for okd 4.15.0-0.okd-2024-02-23-163410..."
Error starting machine: Error in driver during machine start: virError(Code=9, Domain=10, Message='operation failed: Unable to find a satisfying virtiofsd')
Error: Process completed with exit code 1.
```

I had fixed a similar issue on JKube Integration tests in https://github.com/eclipse-jkube/jkube-integration-tests/pull/395

### Expected Behavior
Test OKD Bundle GitHub action should not fail with missing dependency error

### Acceptance Criteria
- [ ] Test OKD Bundle GitHub action is able to start CRC cluster

--- a/.github/workflows/test-okd-bundle.yml
+++ b/.github/workflows/test-okd-bundle.yml
@@ -32,6 +32,8 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt install qemu-kvm libvirt-daemon libvirt-daemon-system
+          # This package may not be present depending on Ubuntu version
+          sudo apt install virtiofsd || true
           sudo usermod -a -G libvirt $USER
       - name: Remove unwanted stuff to free up disk image
         run: |

